src/components/v2/Hero.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+    <section :class="['hero', heroType]" ref="section">
+        <div class="hero-body non-selectable">
+            <div class="container">
+                <h1 class="title" ref="title">
+                    {{title}}
+                </h1>
+                <h2 class="subtitle" ref="subtitle">
+                    {{subtitle}}
+                </h2>
+            </div>
+        </div>
+    </section>
+</template>
+
+<script lang="ts" setup>
+export interface HeroProps {
```

Comment:
Is this exported because Vue demands it, or is it just in case some call site will need it?

Reply:
Looks like we don't need it. Will remove.